src/components/document/TableView/TableView.jsx

```diff
@@ -71,41 +72,41 @@ const TableEntry = (sentence) => {
     }
 
     const SentenceVisibilityScale = (props) => {
-        return <Slider {...props} onChange={setSentenceLength} value={sentenceLength} className="w-20 m-2 justify-center items-center flex"/>
+        return <Slider {...props} onChange={setSentenceLength} value={sentenceLength} className="w-20 m-2 justify-center items-center flex" />
     }
 
     const TableActionButtons = () => {
         return <div className="flex flex-row p-1">
             <img
-                className={redactionResult==Result.Approved ? "opacity-30" : "opacity-100"}
+                className={redactionResult == Result.Approved ? "opacity-30" : "opacity-100"}
                 src={CheckIcon}
-                onClick={() => {setRedactionResult(redactionResult===Result.Approved ? Result.InReview : Result.Approved)}}/>
+                onClick={() => { setRedactionResult(redactionResult === Result.Approved ? Result.InReview : Result.Approved) }} />
             <img
-                className={redactionResult==Result.Rejected ? "opacity-30" : "opacity-100"}
+                className={redactionResult == Result.Rejected ? "opacity-30" : "opacity-100"}
```

Comment:
I’m guessing you used a lint for this, I missed this but it’s supposed to be “===“ here, can you make the change for me